### DISCUSSION TITLE
Are there plans to open source the bun documentation site?

### DISCUSSION THREAD

**Discussion by ikxin** [oven-sh/bun#10495] [General]
It's great to see a javascript runtime with such high performance as bun.

I hope to help more developers understand bun by translating its documentation.

However, I only found the md files in the bun repository, so I can't build a complete documentation site.

I hope the development team can take this suggestion and provide the full source code for the bun documentation site so that developers from more countries and regions can learn about bun and use it!

**Comment by lbonafe-codezi** [comment]
I would love to create documentation with the interface and organization of Bun's documentation, did they use any tools or was it something proprietary?

**Reply by RiskyMH** [reply]
It used to be a custom react site generator, but now its just mintlify which makes it easy to selfhost now!
